controllers/operatorconfigmap_controller.go

```diff
@@ -51,17 +51,17 @@ import (
 var pvcPrometheusRules string
 
 const (
-	operatorConfigMapName = "ocs-client-operator-config"
 	// ClusterVersionName is the name of the ClusterVersion object in the
 	// openshift cluster.
 	clusterVersionName = "version"
 )
 
-// ClusterVersionReconciler reconciles a ClusterVersion object
-type ClusterVersionReconciler struct {
+// OperatorConfigMapReconciler reconciles a ClusterVersion object
+type OperatorConfigMapReconciler struct {
 	client.Client
 	OperatorDeployment *appsv1.Deployment
 	OperatorNamespace  string
+	OperatorConfigMap  *corev1.ConfigMap
```

Comment:
do not store the configmap here, let's deploy configmap as part of the bundle and fetch it in every reconcile.